website/client-old/manifest.json

```diff
@@ -30,9 +30,12 @@
 
             "bower_components/jquery-ui/ui/minified/jquery.ui.core.min.js",
             "bower_components/jquery-ui/ui/minified/jquery.ui.widget.min.js",
+            "bower_components/jquery-ui/ui/minified/jquery.ui.menu.min.js",
```

Comment:
how big is all the jquery ui stuff? Also I think we already have an autocomplete widget, the one we're using in chat, is is necessary to add this one?